[0] setting up environment
2019-09-17 16:31:59.848203: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-09-17 16:31:59.865353: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2099925000 Hz
2019-09-17 16:31:59.870925: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x7f4b4ff17c30 executing computations on platform Host. Devices:
2019-09-17 16:31:59.870989: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
2019-09-17 16:32:00.585125: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x7f4b4ffc74e0 executing computations on platform CUDA. Devices:
2019-09-17 16:32:00.585187: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): Tesla P100-PCIE-16GB, Compute Capability 6.0
2019-09-17 16:32:00.585448: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1433] Found device 0 with properties: 
name: Tesla P100-PCIE-16GB major: 6 minor: 0 memoryClockRate(GHz): 1.3285
pciBusID: 0000:83:00.0
totalMemory: 15.90GiB freeMemory: 15.64GiB
2019-09-17 16:32:00.585471: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2019-09-17 16:32:00.589029: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-09-17 16:32:00.589055: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0 
2019-09-17 16:32:00.589065: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N 
2019-09-17 16:32:00.589221: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 15216 MB memory) -> physical GPU (device: 0, name: Tesla P100-PCIE-16GB, pci bus id: 0000:83:00.0, compute capability: 6.0)
2019-09-17 16:32:09.489909: I tensorflow/stream_executor/dso_loader.cc:152] successfully opened CUDA library libcublas.so.10.0 locally

WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.

=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~
=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~
1287964
[      0       1       2 ... 1287961 1287962 1287963]
[[14.68299961 14.22999954 14.18299961 ... 14.215      14.258
   0.70141297]
 [14.69099998 14.53499985 14.23900032 ... 14.335      14.38
   0.44869809]
 [14.13899994 13.82800007 13.79300022 ... 13.613      13.661
   0.73302806]
 ...
 [11.91100025 11.63199997 11.58600044 ... 11.506      11.528
   1.10468381]
 [ 9.54800034  9.02200031  8.90299988 ...  8.825       8.943
   0.72845937]
 [12.15400028 11.87199974 11.81400013 ... 11.807      11.829
   1.2345698 ]]
68.96957827687106
-1.6096654796396705
[[0.52449938 0.49912038 0.49648724 ... 0.49828004 0.50068908 0.03274445]
 [0.5249476  0.5162078  0.49962464 ... 0.50500295 0.50752404 0.02916387]
 [0.49402219 0.47659864 0.4746378  ... 0.46455341 0.46724258 0.03319239]
 ...
 [0.36920009 0.35356929 0.3509922  ... 0.34651023 0.34774277 0.03845818]
 [0.23681469 0.20734592 0.200679   ... 0.19630911 0.20291998 0.03312766]
 [0.38281399 0.36701511 0.36376572 ... 0.36337355 0.36460608 0.04029847]]
Traceback (most recent call last):
  File "g_PDM_p_all.py", line 510, in <module>
    neural_network.export(save_mod,sess)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow_hub/module.py", line 319, in export
    self._impl.export(path, session)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow_hub/native_module.py", line 586, in export
    self._spec._export(path, variables_saver)  # pylint: disable=protected-access
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow_hub/native_module.py", line 358, in _export
    overwrite=False)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow_hub/tf_utils.py", line 67, in atomic_write_string_to_file
    tf_v1.gfile.Rename(temp_pathname, filename, overwrite)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow/python/lib/io/file_io.py", line 508, in rename
    rename_v2(oldname, newname, overwrite)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow/python/lib/io/file_io.py", line 526, in rename_v2
    compat.as_bytes(src), compat.as_bytes(dst), overwrite, status)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow/python/framework/errors_impl.py", line 528, in __exit__
    c_api.TF_GetCode(self.status.status))
tensorflow.python.framework.errors_impl.AlreadyExistsError: file already exists
srun: error: cp1-p: task 0: Exited with exit code 1
2019-09-17 21:06:22.435224: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-09-17 21:06:22.448359: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2099925000 Hz
2019-09-17 21:06:22.452821: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x7ff3f300a120 executing computations on platform Host. Devices:
2019-09-17 21:06:22.452869: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
2019-09-17 21:06:23.123284: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x7ff3f30b99e0 executing computations on platform CUDA. Devices:
2019-09-17 21:06:23.123344: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): Tesla P100-PCIE-16GB, Compute Capability 6.0
2019-09-17 21:06:23.123610: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1433] Found device 0 with properties: 
name: Tesla P100-PCIE-16GB major: 6 minor: 0 memoryClockRate(GHz): 1.3285
pciBusID: 0000:83:00.0
totalMemory: 15.90GiB freeMemory: 15.64GiB
2019-09-17 21:06:23.123633: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2019-09-17 21:06:23.127265: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-09-17 21:06:23.127290: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0 
2019-09-17 21:06:23.127300: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N 
2019-09-17 21:06:23.127449: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 15216 MB memory) -> physical GPU (device: 0, name: Tesla P100-PCIE-16GB, pci bus id: 0000:83:00.0, compute capability: 6.0)
2019-09-17 21:06:29.524538: I tensorflow/stream_executor/dso_loader.cc:152] successfully opened CUDA library libcublas.so.10.0 locally

WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.

=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~
=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~
1287964
[      0       1       2 ... 1287961 1287962 1287963]
[[14.68299961 14.22999954 14.18299961 ... 14.215      14.258
   0.70141297]
 [14.69099998 14.53499985 14.23900032 ... 14.335      14.38
   0.44869809]
 [14.13899994 13.82800007 13.79300022 ... 13.613      13.661
   0.73302806]
 ...
 [11.91100025 11.63199997 11.58600044 ... 11.506      11.528
   1.10468381]
 [ 9.54800034  9.02200031  8.90299988 ...  8.825       8.943
   0.72845937]
 [12.15400028 11.87199974 11.81400013 ... 11.807      11.829
   1.2345698 ]]
68.96957827687106
-1.6096654796396705
here
Traceback (most recent call last):
  File "T_PDM_p_all.py", line 511, in <module>
    neural_network.export(save_mod,sess)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow_hub/module.py", line 319, in export
    self._impl.export(path, session)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow_hub/native_module.py", line 586, in export
    self._spec._export(path, variables_saver)  # pylint: disable=protected-access
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow_hub/native_module.py", line 358, in _export
    overwrite=False)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow_hub/tf_utils.py", line 67, in atomic_write_string_to_file
    tf_v1.gfile.Rename(temp_pathname, filename, overwrite)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow/python/lib/io/file_io.py", line 508, in rename
    rename_v2(oldname, newname, overwrite)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow/python/lib/io/file_io.py", line 526, in rename_v2
    compat.as_bytes(src), compat.as_bytes(dst), overwrite, status)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/tensorflow/python/framework/errors_impl.py", line 528, in __exit__
    c_api.TF_GetCode(self.status.status))
tensorflow.python.framework.errors_impl.AlreadyExistsError: file already exists
srun: error: cp1-p: task 0: Exited with exit code 1
2019-09-18 01:41:51.346365: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-09-18 01:41:51.355961: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2099925000 Hz
2019-09-18 01:41:51.359555: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x7f502f471fe0 executing computations on platform Host. Devices:
2019-09-18 01:41:51.359605: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
2019-09-18 01:41:51.988104: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x7f502f521890 executing computations on platform CUDA. Devices:
2019-09-18 01:41:51.988151: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): Tesla P100-PCIE-16GB, Compute Capability 6.0
2019-09-18 01:41:51.988392: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1433] Found device 0 with properties: 
name: Tesla P100-PCIE-16GB major: 6 minor: 0 memoryClockRate(GHz): 1.3285
pciBusID: 0000:83:00.0
totalMemory: 15.90GiB freeMemory: 15.64GiB
2019-09-18 01:41:51.988428: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2019-09-18 01:41:51.991491: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-09-18 01:41:51.991514: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0 
2019-09-18 01:41:51.991524: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N 
2019-09-18 01:41:51.991659: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 15216 MB memory) -> physical GPU (device: 0, name: Tesla P100-PCIE-16GB, pci bus id: 0000:83:00.0, compute capability: 6.0)

WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.

=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~
=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~
Traceback (most recent call last):
  File "nu_MDN_p.py", line 499, in <module>
    X_train, y_train, X_test, y_test, classy, params, ymax, ymin, xmax, xmin, xmax_a, xmin_a, train_tinds, test_tinds = GenData_lamost(fileIn = 'lamost_rc_qual.fits')
  File "nu_MDN_p.py", line 86, in GenData_lamost
    al = Table.read(fileIn)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/astropy/table/connect.py", line 52, in __call__
    out = registry.read(cls, *args, **kwargs)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/astropy/io/registry.py", line 508, in read
    fileobj = ctx.__enter__()
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/astropy/utils/data.py", line 193, in get_readable_fileobj
    fileobj = io.FileIO(name_or_obj, 'r')
FileNotFoundError: [Errno 2] No such file or directory: 'lamost_rc_qual.fits'
srun: error: cp1-p: task 0: Exited with exit code 1
2019-09-18 01:41:57.404324: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-09-18 01:41:57.413653: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2099925000 Hz
2019-09-18 01:41:57.416898: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x7f2b1f3dd4b0 executing computations on platform Host. Devices:
2019-09-18 01:41:57.416929: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
2019-09-18 01:41:58.047782: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x7f2b1f48cd70 executing computations on platform CUDA. Devices:
2019-09-18 01:41:58.047828: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): Tesla P100-PCIE-16GB, Compute Capability 6.0
2019-09-18 01:41:58.048080: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1433] Found device 0 with properties: 
name: Tesla P100-PCIE-16GB major: 6 minor: 0 memoryClockRate(GHz): 1.3285
pciBusID: 0000:83:00.0
totalMemory: 15.90GiB freeMemory: 15.64GiB
2019-09-18 01:41:58.048109: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2019-09-18 01:41:58.051384: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-09-18 01:41:58.051408: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0 
2019-09-18 01:41:58.051418: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N 
2019-09-18 01:41:58.051568: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 15216 MB memory) -> physical GPU (device: 0, name: Tesla P100-PCIE-16GB, pci bus id: 0000:83:00.0, compute capability: 6.0)

WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.

=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~
=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~=~
Traceback (most recent call last):
  File "P_MDN_p.py", line 516, in <module>
    X_train, y_train, X_test, y_test, classy, params, ymax, ymin, xmax, xmin, xmax_a, xmin_a, train_tinds, test_tinds, ids = GenData_lamost(fileIn = 'lamost_rc_qual.fits',copy=True)
  File "P_MDN_p.py", line 73, in GenData_lamost
    al = Table.read(fileIn)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/astropy/table/connect.py", line 52, in __call__
    out = registry.read(cls, *args, **kwargs)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/astropy/io/registry.py", line 508, in read
    fileobj = ctx.__enter__()
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/homes/nramachandra/anaconda3/envs/tf_gpu/lib/python3.6/site-packages/astropy/utils/data.py", line 193, in get_readable_fileobj
    fileobj = io.FileIO(name_or_obj, 'r')
FileNotFoundError: [Errno 2] No such file or directory: 'lamost_rc_qual.fits'
srun: error: cp1-p: task 0: Exited with exit code 1
[33005] End job
